js/ui/grid_core/ui.grid_core.state_storing.js

```diff
@@ -175,7 +175,9 @@ module.exports = {
                         columnsController = that.getController("columns"),
                         dataController = that.getController("data"),
                         filterSyncController = that.getController("filterSync"),
-                        scrollingMode = that.option("scrolling.mode");
+                        scrollingMode = that.option("scrolling.mode"),
+                        isVirtualScrollingMode = scrollingMode === "virtual" || scrollingMode === "infinite",
+                        showPageSizeSelector = that.option("pager.visible") && that.option("pager.showPageSizeSelector");
```

Comment:
`that.option("pager.visible")` should be changed to `that.option("pager.visible") === true` because the default value for pager.visible option is "auto". It is my mistake. Also, tests should be changed.